Source/macOS/OIDRedirectHTTPHandler.m

```diff
@@ -76,10 +76,18 @@ - (NSURL *)startHTTPListener:(NSError **)returnError {
       *returnError = error;
     }
     return nil;
-  } else {
+  } else if ([_httpServ hasIPv4Socket]) {
```

Comment:
That would be very(!) much appreciated.

I do know Apple is concerned with IPv6-only networks ([see here](https://developer.apple.com/support/ipv6/)), so I think the feature may be more valuable than you're giving it credit for. IMHO, the lack of this support is a bug with the existing code.